app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module CommentsHelper
+  def rights_to_remove?(comment)
+    current_user && ((comment.user_id == current_user.id) ||
+        current_user.admin? || current_user.moderator?)
+  end
+
+  def paginate_comments_of(commentable)
+    commentable.comments.ordered.page(params[:page]).per(15)
+  end
+
+  def format_date(date)
+    I18n.l date, format: :published
```

Comment:
and actually remove this helper.
use in view what you need, you dont have any logic here
and actually this is not a date but time